JS/Planes/experimentalPlane.js

```diff
@@ -1,68 +1,22 @@
 const Plane = require('./Plane');
```

Comment:
It'd be better to have all files name having the same structure. JS/Planes/experimentalPlane.js  --> JS/Planes/**E**xperimentalPlane.js